README.md

```diff
@@ -11,49 +11,60 @@ This improves the capacity of the plugin to adequately support newer or differen
 7. Test and enjoy!
 
 ## Configuration
 
-The plugin uses the following configuration options. If these are set in the plugin settings UI they override settings configured with code. 
+The plugin menu appears under the Dashboard main menu item. Don't look for it in Settings or Tools, it won't be there :).
 
-There are other options for defining the plugin's functionality, as well. They are:
+The plugin can use the following configuration options in wp-config.php:
 
+| Configuration Parameter |                                                                  Description |                        Default, if any |
+| :---------------------- | ---------------------------------------------------------------------------: | -------------------------------------: |
+| AP_ENABLE               |                         The API Key for AspireCloud (not currently enforced) |                                   none |
+| AP_API_KEY              |                                                          Enable API rewrites |                                   true |
+| AP_HOST                 |                                                              API domain name |                    api.aspirecloud.org |
+| AP_DEBUG                |                                                            Enable Debug Mode |                                  false |
+| AP_DEBUG_TYPES          |                                                      an array of debug modes | array('string', 'request', 'response') |
+| AP_DISABLE_SSL          |                                  Disabled SSL verification for local testing |                                   true |
+| AP_REMOVE_UI            | Disables plugin settings, defaults to config parameters set in wp-config.php |                                  false |
 
-| Configuration Parameter |                                          Description |                        Default, if any |
-| :---------------------- | ---------------------------------------------------: | -------------------------------------: |
-| AP_ENABLE               | The API Key for AspireCloud (not currently enforced) |                                   none |
-| AP_API_KEY              |                                  Enable API rewrites |                                   true |
-| AP_HOSTS                |                                    an array of hosts |           array('api.aspirecloud.org') |
-| AP_DEBUG                |                                    Enable Debug Mode |                                  false |
-| AP_DEBUG_TYPES          |                              an array of debug modes | array('string', 'request', 'response') |
-| AP_DISABLE_SSL          |          Disabled SSL verification for local testing |                                   true |
+NOTE: The AspireUpdate user interface settings _will_ override any plugin options set in wp-config.php unless AP_REMOVE_UI configuration parameter is set to true.
```

Comment:
1) Constants set in wp-config always has priority and user settings wont override it.
2) AP_REMOVE_UI will just remove the UI, not affect anything else.  This is intended to be used where everything is pre configured in wp-config and user configuration not expected.

Reply:
I updated the readme with your notes.